streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/python/PythonGateway.java

```diff
@@ -39,7 +39,7 @@ public PythonGateway() {
 
   public byte[] createStreamingContext() {
     streamingContext = StreamingContext.buildContext();
```

Comment:
I think `streamingContext = StreamingContext.buildContext();` should be `streamContext = StreamContext.buildContext();`

Reply:
Nice catch, Thanks! 
It looks like the IDE refactoring didn't work properly. Fixed with many other classes together.